unison-src/transcripts/merges.output.md

```diff
@@ -112,7 +112,7 @@ We can also delete the fork if we're done with it. (Don't worry, it's still in t
   Note: The most recent namespace hash is immediately below this
         message.
   
-  ⊙ #hpg84g7itp
+  ⊙ #7gjmqmlj0v
```

Comment:
I tracked down what caused this.

The stepManyAt on trunk loses history in sub-namespaces when they're deleted. This new implementation keeps the history intact.

Reply:
/cc @pchiusano I forget whether the old behavior was deliberate, coincidental, or known-undesirable; do you remember?

Reply:
Good catch. Old behavior is wrong, this PR looks more correct.

Deleting a subnamespace should not remove the history of that namespace from its parent. This is important for having merge do the right thing in more cases, since merge will then remember that the child namespace was deleted and not just resurrect the whole thing when there's an unrelated change at the same level.